src/app/util/sqidStatementTable.directive.js

```diff
@@ -94,13 +95,14 @@ function($compile, Properties, dataFormatter, util, i18n) {
 				var hideSomeStatements = (statementGroup.length > hideStatementsThreshold + 1);
 				angular.forEach(statementGroup, function (statement, index) {
 					hasContent = true;
+					var isProposal = (statement.source != 'Wikidata') && (statement.source);
```

Comment:
What is the point of the second part of this condition? Is this is needed, shouldn't you check that statement.source is set before comparing it to 'Wikidata'?